simpledotorg/simple-server
# YAMLize help screen

**Story card:** [ch2968](https://app.clubhouse.io/simpledotorg/story/2968/convert-help-locales-to-yml)

## Because

The current HTML locale format of the help screens makes it difficult to:
* Introduce dynamic elements to the help page
* Introduce country-specific elements to the help page
* Manage translations in Transifex when large structural changes are made to the help screen

## This addresses

Extracting all Help text into a YAML-based locale file.

## Test instructions

* Turn on feature flag `yaml_help`
* Ensure the Help screen looks good and is formatted correctly in English only.

## After this PR

After this PR is merged, next steps will be:
* Making sure the new locale keys are imported into Transifex automatically
* Making sure most of the new keys are auto-translated (we have turned on the "Memory Lookup" feature in Transifex)
* Ask GPI to fill in the missing translations
* Accept the PRs that add the new locale keys to all other languages
* Turn on the feature flag
* Once confident, remove the feature flag and old HTML locales completely.
